web/src/main/resources/application.yml

```diff
@@ -31,3 +31,8 @@ spring:
           time_zone: UTC
           lob:
             non_contextual_creation: true
+auth:
+  tokenSecret: 928d4aef-6452-4ca1-8e4e-7dabad510423
```

Comment:
Ở đây không dùng kiểu hex 256 à

Reply:
Cái này chỉ là secret key mà, hex 256 là sao @leanhdung0505

Reply:
Kiểu hexan 256-bit

Reply:
Ý là mục đích để chi á m

Reply:
Một kiểu dữ liệu của secret key, t thấy họ hay dùng nên t hỏi

Reply:
Ví dụ thử 1 cái xem m

Reply:
![image](https://github.com/LoadPulse/loadpulse-be/assets/95488550/c33599cb-0ec0-4a94-b9d6-f435b81b60d2)
Đây m

Reply:
M đưa 1 cái chuỗi 256 bit nớ xem chớ trên ảnh t thấy cái nớ string mà

Reply:
![image](https://github.com/LoadPulse/loadpulse-be/assets/95488550/42e58518-02e7-458e-a714-a8cd7d92c9e7)